test/operators/dedicatedadmin.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 // timeout is the duration in minutes that the polling should last
```

Comment:
This is no longer true - globalPollingTimeout is now the time in seconds

Reply:
Good catch. Fixed.